api/types/installers/installer.sh.tmpl

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+( flock -n 9 || exit 1;
+  if test -f /usr/local/bin/teleport; then
+          exit 0
+  fi
+
+  distro_id="$(awk -F= '$1 == "ID" { print tolower($2) }' /etc/os-release | xargs echo)"
+  if [ "$distro_id" = "debian" ] || [ "$distro_id" = "ubuntu" ]; then
+          sudo curl https://deb.releases.teleport.dev/teleport-pubkey.asc \
+                   -o /usr/share/keyrings/teleport-archive-keyring.asc
+          echo "deb [signed-by=/usr/share/keyrings/teleport-archive-keyring.asc] https://deb.releases.teleport.dev/ stable main" \
+                  | sudo tee /etc/apt/sources.list.d/teleport.list > /dev/null
+
+          sudo apt-get update
+          sudo apt-get install -y teleport jq
+  elif [ "$distro_id" = "amzn" ] || [ "$distro_id" = "rhel" ]; then
+          sudo yum-config-manager --add-repo 'https://rpm.releases.teleport.dev/teleport.repo'
+          sudo yum install -y teleport jq
+  fi
```

Comment:
Couple notes:

1. This should use the new APT repositories (https://goteleport.com/docs/installation/#linux). New RPM repos are also coming out this week, so we'll need to update yum instructions also.
2. I don't see where we specify the Teleport version that is being installed. We want to install the same version that auth server is running as we discussed before via an install parameter, right?